gh-runners/windows/main.tf

```diff
@@ -6,7 +6,7 @@ resource "metal_device" "windows-lcow" {
   project_id       = var.METAL_PROJECT_ID
   hostname         = "windows-lcow-gh-runner"
   operating_system = "windows_2019"
-  facilities       = ["dfw2", "iad2"]
+  facilities       = ["da11", "dfw2", "iad1", "dc13"]
```

Comment:
@jromero I just made this change because it seemed like you had done so a few days back, when you ran the build. I admit that I'm not sure what this does

Reply:
These are data centers that equinix uses. `iad2` was "decommissioned" in a way that it was no longer available. I saw an attempt fail while trying to assign it to that facility so I looked at their [latest info](https://metal.equinix.com/developers/docs/locations/facilities/#facilities-quick-reference) and added a few replacements.